kstreams/streams.py

```diff
@@ -316,19 +292,40 @@ async def __anext__(self) -> ConsumerRecord:
 
         try:
             return await self.getone()
+            # return await self.func(cr)
         except errors.ConsumerStoppedError:
             raise StopAsyncIteration  # noqa: F821
 
 
+class UdfHandler:
```

Comment:
Could you add a docstring to this one, what is Udf? From reading the code, I can guess it's doing some of the magic for the DI, right?

Reply:
Yes. This class is in change of calling the user defined function in a proper way.